src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        return new Ball(colorSupplier.getRandomColor(), random.nextInt(100));
```

Comment:
The number 100 used in `random.nextInt(100)` is a magic number. It should be defined as a constant field with a clear name to indicate its purpose and to make the code more maintainable.